src/js/common/schemaform/definitions/dateRange.js

```diff
@@ -3,17 +3,23 @@ import * as date from './date';
 
 import commonDefinitions from 'vets-json-schema/dist/definitions.json';
 
-export const schema = commonDefinitions.dateRange;
+export const schema = {
+  type: 'object',
+  properties: {
+    from: commonDefinitions.date,
+    to: commonDefinitions.date
+  }
+};
```

Comment:
Ah, OK--I saw $ref used all over the place so figured we were generally for it.

Aside: it might be useful to start a vets-website FE style guide that starts to capture any opinions we as a team develop, esp. regarding rjsf but anything else we use heavily, esp. if there are no external style guides we're conforming to. It could start as just a few thoughts and evolve as things become stable.